src/app/containers/MediaPageMain/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,886 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Media Page Main snapshots should match scaffold snapshot 1`] = `
+.c2 {
+  color: #3F3F42;
+  font-family: ReithSerif,Helvetica,Arial,sans-serif;
+  margin: 0;
+  padding: 2rem 0;
+  font-weight: 500;
+}
+
+.c0 {
+  margin: 0 auto;
+  padding-bottom: 2rem;
+}
+
+.c1 {
+  grid-column: 1 / span 6;
+}
+
+@media (min-width:37.5rem) {
+  .c2 {
+    padding: 2.5rem 0;
+  }
+}
+
+@media (max-width:37.4375rem) {
+  .c0 {
+    grid-column-gap: 0.5rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c0 {
+    grid-column-gap: 1rem;
+  }
+}
+
+@media (max-width:62.9375rem) {
+  .c0 {
+    grid-template-columns: repeat(6,1fr);
+  }
+}
+
+@media (min-width:63rem) and (max-width:80rem) {
+  .c0 {
+    grid-template-columns: 1fr repeat(8,minmax(0,6.75rem)) 1fr;
+    max-width: 45.5rem;
+  }
+}
+
+@media (min-width:80rem) {
+  .c0 {
+    grid-template-columns: 1fr repeat(20,minmax(0,2.95rem)) 1fr;
+    max-width: 46.4rem;
+  }
+}
+
+@supports (display:grid) {
+  .c0 {
+    display: grid;
+    max-width: initial;
+    margin: initial;
+  }
+}
+
+@media (max-width:25rem) {
+  .c1 {
+    padding: 0 0.5rem;
+  }
+}
+
+@media (min-width:25rem) and (max-width:62.9375rem) {
+  .c1 {
+    padding: 0 1rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c1 {
+    grid-column: 1 / span 5;
+    max-width: 83.33%;
+  }
+}
+
+@media (min-width:63rem) and (max-width:80rem) {
+  .c1 {
+    grid-column: 3 / span 5;
+    max-width: 37.75rem;
+  }
+}
+
+@media (min-width:80rem) {
+  .c1 {
+    grid-column: 6 / span 10;
+    max-width: 38.5rem;
+  }
+}
+
+@supports (display:grid) {
+  .c1 {
+    max-width: initial;
+  }
+}
+
+<main
+  role="main"
+>
+  <div
+    className="c0"
+  >
+    <div
+      className="c1"
+    >
+      <h1
+        className="c2"
+      >
+        Default Brand Name
+         - Live Radio
+      </h1>
+      <ul>
+        <li>
+          <strong>
+            Service
+          </strong>
+          : 
+          amharic
+        </li>
+        <li>
+          <strong>
+            Brand
+          </strong>
+          : 
+          bbc_amharic_radio
+        </li>
+        <li>
+          <strong>
+            MediaId
+          </strong>
+          : 
+          liveradio
+        </li>
+      </ul>
+    </div>
+  </div>
+</main>
+`;
+
+exports[`Media Page snapshots should match scaffold snapshot 1`] = `
```

Comment:
Looks like `src/app/containers/MediaPageMain/__snapshots__/index.test.jsx.snap` is containing the ``Media Page` snapshot as well? 

Try running `npm run test:unit -- -u`.

Reply:
At a closer look it might be because you are running `import sampleProps from '../MediaPage/index.test';` that jest is actually re-triggering the test in the imported file so the output is printed in this test as well.

Reply:
Tried removing the import locally and duplicating the `liveRadioScaffoldProps` config and it fixed the duplicate test. 

Haha Jest is weird 🙃